modules/agent-install-networking.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_content-type: CONCEPT
+[id="agent-install-networking_{context}"]
+= About networking
+
+During the initial boot, the machines require an IP address configuration that is set either through a Dynamic Host Configuration Protocol(DHCP) server or statically by
+selecting the below options. If you provide the IP address for the `rendezvousIP` field through the DHCP option, then ensure that it is for the machine's IP address that is going to be used for deployment.
+This IP address is required for a node to be identified as ** node 0 **. A ** node 0 ** runs the required services for the Agent-based Installer.
+
+.DHCP
+
+.. Preferred Method: `install-config.yaml` and `agent.config.yaml`
```

Comment:
In the preview, I think the a) list item is confusing here because it leads be to thing there will be a b), and there's not. It makes more sense to use it below with Static Networking.

Reply:
@skrthomas  I will just remove the a) thanks.